kyo-core/shared/src/main/scala/kyo/Resource.scala

```diff
@@ -35,56 +35,6 @@ import kyo.kernel.ContextEffect
 sealed trait Resource extends ContextEffect[Resource.Finalizer]
 
 object Resource:
-    /** Represents a finalizer for a resource. */
```

Comment:
There are no changes in this section of the code. I've just moved it to the end of the file so the public APIs appear first in the file to improve readability.